values.yaml

```
@@ -93,6 +93,43 @@ injector:
       exitOnRetryFailure: true
       staticSecretRenderInterval: ""
 
+  # Used to enable a livenessProbe for the pods
```

Very picky nit, as the liveness probe is always enabled.

```suggestion
  # Used to configure a livenessProbe for the pods
```